### PR TITLE
update inkscape to 0.91-1

### PR DESCRIPTION
**Pull request by jconley**

### CODE REVIEW

File: Casks/inkscape.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 cask :v1 => 'inkscape' do
-  version '0.48.5-2'
-  sha256 '72191861ee19a4e047d9084c7181a5ccf6e89d9b4410e197a98c2e1027e65e72'
+  version '0.91-1'
+  sha256 '66c9af880f87d5b64006391fc8dd2d3f546d93d6d91235eba7856deacba29862'
 
-  # sourceforge.net is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/inkscape.rb]: This comment needs to remain, substituting `sourceforge.net` with `fastly.net`.

Comment (jconley) [inline Casks/inkscape.rb]: the latest in sourceforge is only 0.91pre4

Comment (vitorgalvao) [inline Casks/inkscape.rb]: Yes, I’m talking about the comment, not the new link itself.

Comment (jconley) [inline Casks/inkscape.rb]: re-added

Comment (vitorgalvao) [inline Casks/inkscape.rb]: Thank you.

**Pull request merged**